src/conic_algorithm.jl

```diff
@@ -1879,7 +1879,10 @@ function clean_zeros!{N}(m, data::Array{Float64,N})
 
     if max_nz > m.cut_zero_tol
         if max_nz/min_nz > 1e7
-            warn("Numerically unstable dual vector encountered (maxabs = $max_nz, minabs = $min_nz)\n")
+            if logs[:n_unst_dual] == 0
+                warn("The conic solver is returning numerically unstable conic duals\n")
```

Comment:
Misleading warning, seems to put blame on the conic solver when it may not be doing anything wrong.